www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR NON-INFRINGEMENT.
+
+1. Limitation of Liability
+
+  1. TO THE MAXIMUM EXTENT PERMITTED BY APPLICABLE LAW, GRATIPAY SHALL NOT BE
+     LIABLE FOR ANY INDIRECT, INCIDENTAL, SPECIAL, CONSEQUENTIAL OR PUNITIVE
+DAMAGES, OR ANY LOSS OF PROFITS OR REVENUES, WHETHER INCURRED DIRECTLY OR
+INDIRECTLY, OR ANY LOSS OF DATA, USE, GOOD-WILL, OR OTHER INTANGIBLE LOSSES,
+RESULTING FROM (i) YOUR ACCESS TO OR USE OF OR INABILITY TO ACCESS OR USE THE
+SERVICE; (ii) ANY CONDUCT OR CONTENT OF ANY THIRD PARTY ON THE SERVICE,
+INCLUDING WITHOUT LIMITATION, ANY DEFAMATORY, OFFENSIVE OR ILLEGAL CONDUCT OF
+OTHER PARTICIPANTS OR THIRD PARTIES; (iii) ANY CONTENT OBTAINED FROM THE
+SERVICE; OR (iv) UNAUTHORIZED ACCESS, USE OR ALTERATION OF YOUR TRANSMISSIONS
+OR CONTENT.  IN NO EVENT SHALL THE AGGREGATE LIABILITY OF GRATIPAY OR ITS
+AFFILIATES EXCEED THE GREATER OF ONE HUNDRED U.S. DOLLARS (U.S. $100.00) OR THE
+AMOUNT YOU PAID GRATIPAY, IF ANY, IN THE PAST SIX MONTHS FOR THE SERVICE GIVING
+RISE TO THE CLAIM.
+
+1. Indemnification
+
+  1. You agree to defend, indemnify and hold harmless Gratipay and its
+     officers, directors, employees, contractors and agents from and against
+any and all claims, causes of action, damages, obligations, losses,
+liabilities, costs or debt, and expenses (including attorneys' fees and costs)
+and all amounts paid in settlement arising from or relating to use of the
+Service, breach of these Terms or violation of any laws. Gratipay may assume
+the exclusive defense and control of any matter for which you have agreed to
+indemnify Gratipay and you agree to assist and cooperate with Gratipay in the
+defense or settlement of any such matters.
+
+1. Miscellaneous Terms
+
+  1. Assignment: You may not assign your rights and obligations under these
+     Terms to any other person. In the event of a merger, sale, or acquisition
+of substantially all of Gratipay's assets, Gratipay may assign its rights and
+obligations under these Terms to its successor.
+
+  1. Modifications to this Agreement: Gratipay may occasionally revise these
+     Terms. You can find the most recent version at
+http://gratipay.com/about/terms/. If we determine, in our sole discretion, that
+a change to these Terms is material, we will notify you of the change through
+the Service or by emailing you at the address associated with your account. By
+using the Service after a new version of these Terms becomes effective, you
+thereby agree to be bound by the revised Terms.
+
+  1. Choice of Law: These Terms and the relationship between you and Gratipay
+     shall be governed by the laws of the State of Pennsylvania without regard
+to its conflict of law provisions. Any and all claims, causes of action, or
+disputes (regardless of theory) arising out of or relating to the Service,
+shall be brought exclusively in the courts located in Allegheny County,
```

Comment:
Gratipay is actually headquartered in Beaver County, not Allegheny. Did we intentionally choose Allegheny over Beaver?